eslint.config.mjs

```diff
@@ -46,17 +46,6 @@ export default tseslint.config(
             globals: {
                 ...globals.es2021,
             },
-
-            parserOptions: {
-                projectService: {
-                    allowDefaultProject: [
-                        // I'm not sure why these files aren't picked up... :\
-                        'packages/@lwc/module-resolver/scripts/test/matchers/to-throw-error-with-code.ts',
-                        'packages/@lwc/module-resolver/scripts/test/matchers/to-throw-error-with-type.ts',
-                        'packages/@lwc/module-resolver/scripts/test/setup-test.ts',
-                    ],
```

Comment:
Now they're picked up and we're also not sure why! 😄

Reply:
Probably this: https://devblogs.microsoft.com/typescript/announcing-typescript-5-7/#searching-ancestor-configuration-files-for-project-ownership